### DISCUSSION TITLE
401 even after refreshed token

### DISCUSSION THREAD

**Discussion by fatshotty** [trakt/trakt-api#318] [Q&A] [answered]
Hello there,
I created my app and get client_id and client_secret.
Now I'm testing API via trakt apiary website but I'm getting 401 while invoking the `/calendars/my/movies` . I correctly set the `bearer token` and the `client_id` , because other API url is working fine.
Also `/calendars/my/tvshows` returns the same 401 error.

Just to be sure, I also tried with the trakt.tv nodejs library setting all data and tokens, but 401 is still there :/

Any idea?
thanks

**Comment by tysonkerridge** [comment, marked as answer]
Would you be able to include the headers of the request and/or anything else that might help to see what you're sending vs what you're receiving?

**Reply by fatshotty** [reply]
thanks for reply.
I created a new app and used new `client_id` and `client_secret` . It is working now.

thanks

**Reply by rectifyer** [reply]
That’s weird, the 401 is strictly based on the OAuth user so creating a new app shouldn’t matter for that error. My guess is the OAuth credentials expired and you just needed to re-authorize your user account.

**Reply by fatshotty** [reply]
thanks
already tried to re-new my account and my access token...
401 was still there :/
I created a new app and got new `client_id` and `client_secret` . Now it is working
